src/plugins/updater/index.ts

```diff
@@ -268,14 +270,19 @@ async function downloadMonika(
   const filename = `monika-v${remoteVersion}-${platformName}-x64`
   const downloadUri = `https://github.com/hyperjumptech/monika/releases/download/v${remoteVersion}/${filename}.zip`
   log.info(`Updater: download from ${downloadUri}`)
-  const { data: downloadStream } = await sendHttpRequest({
+  const resp = await sendHttpRequestFetch({
     url: downloadUri,
     responseType: 'stream',
   })
 
-  const { data: checksum }: { data: string } = await sendHttpRequest({
+  const { data: downloadStream } = (await resp.json()) as { data: Stream }
```

Comment:
I think we can get stream from `resp.body` instead of `resp.json()` ?

ref https://developer.mozilla.org/en-US/docs/Web/API/Streams_API/Using_readable_streams